Header header
string gnss_status
string ins_status
float32 dt
float64 lat
float64 lon
float64 alt
float32 eastPos
float32 northPos
float32 upPos
float32 roll
float32 pitch
float32 azimuth
float32 roll_inc
float32 pitch_inc
float32 azi_inc
float32 east_vel
float32 north_vel
float32 up_vel
float32 x_vel
float32 y_vel
float32 z_vel
float32 eastPos_std
float32 northPos_std
float32 upPos_std
float32 roll_std
float32 pitch_std
float32 azi_std
float32 eastVel_std
float32 northVel_std
float32 upVel_std
